packages/react/src/table/Table.tsx

```diff
@@ -1,9 +1,34 @@
-import { PropsWithChildren, forwardRef } from 'react';
+import {
+	PropsWithChildren,
+	forwardRef,
+	useLayoutEffect,
+	useRef,
+	useState,
+} from 'react';
 import { Box } from '../box';
-import { boxPalette } from '../core';
+import { mergeRefs } from '../core';
 import { TableContext } from './TableContext';
+import { useScrollerContext } from './ScrollerContext';
+
+const MAX_FIXED_COLUMNS_FROM_LEFT = 1;
```

Comment:
This allows us to use a future-proof API of `frozenColumns?: number[];` instead of something like `isFirstColFrozen`, `isLastColFrozen`.